webroot/js/main.js

```diff
@@ -39,12 +37,16 @@ function doApiCallWithBody(method, url, body, outputDiv, errorDiv) {
             var pretty = JSON.stringify(JSON.parse(text),null,2);
             $(outputDiv).html(pretty);
         },
-        error: function (err) {
-            $(errorDiv).html('Error: ' + err.status + ': ' + (isJsonString(err.responseText) ? JSON.parse(err.responseText).message : (err.responseText ? err.responseText : err.statusText)));
-        }
+        error: function (err) { standardErrorCallback(err, outputDiv) }
     });
 }
 
+function errorCallback(err) { standardErrorCallback(err, '#errorOutput') }
+
+function standardErrorCallback(err, errorDiv) {
+    $(errorDiv).html('Error: ' + err.status + ': ' + (isJsonString(err.responseText) ? JSON.parse(err.responseText).message : (err.responseText ? err.responseText : err.statusText)));
```

Comment:
Here and elsewhere, prefer `text()` to `html()` if we're just setting text.

Reply:
switched to text()